bin/vault/usage.py

```diff
@@ -229,7 +275,21 @@ def _resolve_path(path: T.Path) -> T.Path:
 
             # Resolve all paths
             parsed.files = [_resolve_path(path) for path in parsed.files]
-
+        
+        if "fofn" in parsed and parsed.fofn is not None:
+
+            def _create_fofn_generator(fofn: T.Path) -> T.Iterator[T.Path]:
+                # Check: Any problems with yield from within a context manager?   
+                with open(fofn) as file:
+                    while file_line := file.readline():
+                        filepath = T.Path(file_line.rstrip())
+                        resolved_path = filepath.resolve()
```

Comment:
I agree with Michael that it would be good to re-use _resolve_path somehow.

Reply:
Cool, yeah, I agree